conf/sample/networkservice-daemonset.yaml

```diff
@@ -28,6 +29,10 @@ spec:
               mountPath: /var/lib/kubelet/device-plugins
             - name: nsm-socket
               mountPath: /var/lib/networkservicemesh
+            - name: run
+              mountPath: /var/run
+            - name: sys
+              mountPath: /sys
```

Comment:
Why are we adding these very broad mounts to the NSM container?

Reply:
My understanding that dataplanes are orchestrator-agnostic, and does not know anything about k8s (and probably) nothing about docker. If my understanding is correct, it's NSM responsibility to provide pids/linux namespaces of pods to be interconnected down to dataplane. In order to get conainer pids we have to have access /var/run/docker.pid and /sys, I'll check if /sys access can be narrower, but anyway either dataplane of NSM should have access to these paths. Please suggest

Reply:
OK... I'm not sure whey we need to be able to get the docker PIDs... what are we doing with them?

Reply:
OK... I think I get the pid thing, but we are getting the netns from the Pid via the netns lib... which did not appear to require mounting /sys before... so I'm wondering we we need it now?